mgmt/ProxyConfig.h

```diff
@@ -117,15 +117,31 @@ ConfigScheduleUpdate(Ptr<ProxyMutex> &mutex)
   return 0;
 }
 
+template <typename UpdateClass>
```

Comment:
BTW. I still need to test this. I pushed this so that I can access this from home